x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/content.tsx

```diff
@@ -60,6 +68,14 @@ export function ContentPanel(props: ContentPanelProps) {
       );
     case 'policies':
       return <PackagePoliciesPanel name={name} version={version} />;
+    case 'custom':
+      return (
+        (CustomView && (
+          <ExtensionWrapper>
+            <CustomView />
+          </ExtensionWrapper>
+        )) || <Redirect to={getPath('integration_details', { pkgkey: pkgkey ?? '' })} />
+      );
```

Comment:
Mind breaking this up multiple statements? It took me a bit to unpack this at first, and I still got it wrong :)

Also, at least by reading, "get url for integrations_details with key empty string" seems odd. e.g. `/app/fleet#/integrations/detail/`. I think it works in the app, but let's explicitly send the user to named/working view like 'integrations_all`

Perhaps something like:
```suggestion
        const redirectUrl = pkgkey
          ? getPath("integration_details", { pkgkey })
          : getPath("integration_all");
        return CustomView ? (
          <ExtensionWrapper>
            <CustomView />
          </ExtensionWrapper>
        ) : (
          <Redirect to={redirectUrl} />
        );
```

Reply:
Yeah, Agreed. This is condition check because I'm trying to grab the value from the current URL. I'm actually going to change to build it manually based on Component's input props (we have what we need). I will also look to see if there is a utility function that does that from a centralized location (I seem remember such utility, but maybe it was server side).